test/test-connector.js

```diff
@@ -117,7 +117,8 @@ describe('swagger connector', function() {
     describe.skip('models with remotingEnabled', function() {
       let ds;
       before(function(done) {
-        ds = createDataSource('test/fixtures/petStore.json', false, true);
+        ds = createDataSource('test/fixtures/petStore.json',
+          {validate: false, remotingEnabled: true});
```

Comment:
> do you want me to fix it now, not sure if it's okay to update PR after approved

It's ok to update the PR after it was approved when it's a tiny change like this one. We can land your pull request even as it is now, not a big deal.